Xamarin.Essentials/Types/SensorSpeed.uwp.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Xamarin.Essentials
+{
+    static class SensorSpeedExtensions
+    {
+        internal static uint ToPlatform(this SensorSpeed sensorSpeed)
```

Comment:
Shouldn't this not return `8`, `22` & `33`.

Reply:
For the compass, yes it is, but not for other sensors as the documentation don't specify any of it, so we should keep all three platforms the same.

Reply:
Ok sorry, then I probably misunderstood OrientationSensor, magnetometer, gyroscope & accelerometer.

Reply:
Yeah, so for instance accelerometer: https://docs.microsoft.com/en-us/uwp/api/windows.devices.sensors.accelerometer.reportinterval

Where as compass has specifics to use: https://docs.microsoft.com/en-us/uwp/api/windows.devices.sensors.compass.reportinterval

So just restructuring to make them all the same across devices if no info is available.